torch/lib/c10d/CMakeLists.txt

```diff
@@ -128,10 +128,6 @@ endif()
 if(MPI_FOUND)
   target_include_directories(c10d PUBLIC ${MPI_INCLUDE_PATH})
   copy_header(ProcessGroupMPI.hpp)
-  file(WRITE ${CMAKE_BINARY_DIR}/include/c10d/mpi_include_path "")
```

Comment:
you removed this, but in the torch/CMakeLists.txt you haven't included MPI dirs or c10d dirs while compiling the csrc/distributed bits

Reply:
now that we're inside cmake, it's taken care of by modern cmake targets. This was only needed because that didn't work across the cmake/setup.py boundary